contents/handbook/product/metrics.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Product metrics
+sidebar: Handbook
+showTitle: true
+---
+
+As we improve our Product, it's key to have a guiding metric that we can all at Product & Engineering rally behind and measure our progress against. For the Product, this metric is "SNAZZY NAME HERE". The metric can be tracked in the [product dasboard][dashboard] and is reported weekly at PostHog News.
```

Comment:
Let's commit this it with a proposed "Snazzy name" which we can debate in the PR.

I propose two options to get the discussion started: 

- **Meaningful Insight Analyzed** - since this is effectively the definition of the metric
- **Discoveries** - We could rebrand this to be the definition of the Discoveries metric - where we currently use discovered learnings